.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+# Description
+
+<!-- describe the changes you PR introduces here -->
+
+# Issues
+
+<!-- link related issues here -->
+
+# Checklist
+
+<!-- take care of the default items before creating your PR and be prepared to add more items -->
+
+* [ ] I updated the version in Chart.yaml
+* [ ] I added to the docs and updated them using  `pre-commit run -a`
+* [ ] CI is currently green and this is ready for review
+* [ ] I am ready to test changes after they are applied and released
```

Comment:
Shouldnt the changes be tested before applying them?

Reply:
They should be tested through GitHub Actions before merging but you also need to test rolling out the changes after the merge (aka don't update things like kps w/o being ready to roll out the change to at least one (dev) env).